java/de/yessoft/android/mohammad/application/app.java

```diff
@@ -0,0 +1,58 @@
+package de.yessoft.android.mohammad.application;
```

Comment:
please remove mohammad from the package name of all classes

Reply:
OK, i used it to be able to create the app in the firebase console, why i would not clone the repo? because i need sha1 code to be able to add Notifications and Authentication, Which means i need to do this in my Laptop or u need to provide me such a code :) 👍